Q/Qt/build_tarballs.jl

```diff
@@ -47,15 +47,15 @@ case "$target" in
             -skip qtwinextras -fontconfig
 #        ln -s /lib64/libc.so.6 /lib64/libc.so
```

Comment:
```suggestion
```

Reply:
I thought I would leave it in there should it come up in the future.

Reply:
Having to do that feels mostly wrong to me :slightly_smiling_face: